src/Components/Blazor/testassets/Wasm.Authentication.Client/wwwroot/css/open-iconic/FONT-LICENSE

```diff
@@ -0,0 +1,86 @@
+SIL OPEN FONT LICENSE Version 1.1
```

Comment:
@Pilchie should these licenses be included in the https://github.com/dotnet/aspnetcore/blob/blazor-wasm/THIRD-PARTY-NOTICES.txt file?

More generally, is open iconic an approved dependency?

Reply:
This is straight from our templates

Reply:
Then it should already be in our 3PN from when we added it to templates, right?

Reply:
This is gone from the repo, so no longer applies (for the record)